instrumentation/logback/logback-appender-1.0/library/README.md

```diff
@@ -60,3 +60,46 @@ The following demonstrates how you might configure the appender in your `logback
 
 In this example Logback log events will be sent to both the console appender and
 the `OpenTelemetryAppender`.
+
+In order to function, `OpenTelemetryAppender` needs access to an `OpenTelemetry` instance. This can
+be programmatically set during application startup as follows:
+
+```java
+import ch.qos.logback.classic.LoggerContext;
+import io.opentelemetry.instrumentation.logback.OpenTelemetryAppender;
+import io.opentelemetry.sdk.OpenTelemetrySdk;
+
+public class Application {
+
+  public static void main(String[] args) {
+    OpenTelemetrySdk openTelemetrySdk = // Configure OpenTelemetrySdk
+
+    // Get LoggerContext, get OpenTelemetryAppender from Root Logger, and call setOpenTelemetrySdk(...)
+    LoggerContext loggerContext = (LoggerContext) LoggerFactory.getILoggerFactory();
+    ((OpenTelemetryAppender)
+        loggerContext.getLogger(Logger.ROOT_LOGGER_NAME).getAppender("OpenTelemetryAppender"))
```

Comment:
I think it'd probably be safer for copy-pasteable code to use the `iteratorForAppenders()` instead of `getAppender()` with a concrete name -- there's no guarantee the appender will be named `OpenTelemetryAppender` after all.

Reply:
Missed that method. Much better!